e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Customization', () => {
+    let config;
+
+    before(() => {
+        // Get config
+        cy.apiGetConfig().then((response) => {
+            config = response.body;
+        });
+
+        // # Login as sysadmin and visit customization system console page
+        cy.apiLogin('sysadmin');
+        cy.visit('/admin_console/site_config/customization');
+    });
+
+    it('SC20330 - Can change Privacy Policy Link setting', () => {
+        const contents = ['The URL for the Privacy link on the login and sign-up pages. If this field is empty, the Privacy link is hidden from users.'];
+
+        // * Verify that setting is visible and matches text content
+        cy.findByTestId('SupportSettings.PrivacyPolicyLinklabel').should('be.visible').and('have.text', 'Privacy Policy Link:');
```

Comment:
This failed as it needs to be in in the viewing window.
```suggestion
        cy.findByTestId('SupportSettings.PrivacyPolicyLinklabel').scrollIntoView().should('be.visible').and('have.text', 'Privacy Policy Link:');
```